### PR TITLE
MetaGrowler 0.5.2 extensible monitor using Growl for OSX

### PR DESCRIPTION
**Pull request by BenjaminHCCarr**
*If there’s a checkbox you can’t complete for any reason, that's okay, just explain in detail why you weren’t able to do so.*

After making all changes to the cask:

- [X] `brew cask audit --download {{cask_file}}` is error-free.
- [X] `brew cask style --fix {{cask_file}}` reports no offenses.
- [X] The commit message includes the cask’s name and version.

Additionally, **if adding a new cask**:

- [X] Named the cask according to the [token reference].
- [X] `brew cask install {{cask_file}}` worked successfully.
- [X] `brew cask uninstall {{cask_file}}` worked successfully.
- [X] Checked there are no [open pull requests] for the same cask.
- [X] Checked that the cask was not already refused in [closed issues].

---

name 'MetaGrowler: An extensible program for monitoring various activities and events that can occur on your system using Growl'
homepage 'http://buttered-cat.com/products/view/MetaGrowler'

### CODE REVIEW

File: metagrowler.rb [hunk 1]
```diff
@@ -0,0 +1,10 @@
+cask 'metagrowler' do
+  version '0.5.2'
+  sha256 'dc1ae19c1dd7a2af6fd2346911d93109e72b8a59b847341b35d42f36baab5657'
+
+  url "https://buttered-cat.com/downloads/get/12/MetaGrowler_#{version}.zip"
+  name 'MetaGrowler: An extensible program for monitoring various activities and events that can occur on your system using Growl'
```
Comment (miccal) [inline metagrowler.rb]: Please change this to just `name 'MetaGrowler'`.

File: quickboot.rb [hunk 2]
```diff
@@ -0,0 +1,10 @@
+cask 'quickboot' do
+  version '1.1-92'
+  sha256 '5119e1113949baae165efc726b757d58a887743f1dc6ae8d03a978be7b5da4a3'
+
+  url "https://buttered-cat.com/downloads/get/4/QuickBoot-#{version}.zip"
+  name 'QuickBoot is a handy utility to make it easier to reboot into your Boot Camp windows drive, or even another OS X install'
```
Comment (miccal) [inline quickboot.rb]: Please change this to just `name 'QuickBoot'`.

**Comment by BenjaminHCCarr** [comment]
I broke my branch/patch tree, somehow ended up with two casks in one pull. I'll pull the correct quickboot separately.

**Comment by MikeMcQuaid** [comment]
@vitorgalvao @BenjaminHCCarr This was merged into the tap root rather than the Casks directory so Homebrew detects it as a formula, breaks and refuses to tap `caskroom/cask` any more.

CC @reitermarkus @jawshooah as this is pretty urgent.

**Comment by jawshooah** [comment]
Done in https://github.com/caskroom/homebrew-cask/commit/99e75b1c306666f4bcebb07ffdda047d72240dfc.

**Comment by vitorgalvao** [comment]
We should really fix this. I was under the impression `audit` (or Travis, at least) used to detect this.

**Pull request merged**